index.html

```diff
@@ -36,42 +36,36 @@
         </nav>
     </header>
     <main>
-        <section class="bio"><h1>
-        <figure class="mugshot">
-            <a target="_blank" href="images/mugshot.jpg">
-           <img src="images/mugshot.jpg" alt="mugshot">
-           <figcaption>mugshot</figcaption>
-        </figure>
-        </a>
```

Comment:
When I said to put the `<figure>` element after the `<h1>` element I mean after the element as a whole, not nested inside the element.
There is also a problem here with your `<a>` element which doesn't get closed before its parent element `<figure>` is closed